### PR TITLE
add introduction

### PR DESCRIPTION
**Pull request by pwright**

### CODE REVIEW

**Comment by jasonmadigan** [comment]
@finp I'd recommend passing this wording on to integreatly-leaders tbh. There's a few terms included that I'm not sure of the fit (and they get into upstream naming)

**Comment by pwright** [comment]
@jasonmadigan  will do, btw the mention of k8s came from https://www.openshift.com/products/dedicated/

**Comment by pwright** [comment]
closed in favor of https://github.com/integr8ly/user-documentation/pull/16

File: docs/content/rn-intro-con.adoc [hunk 1]
```diff
@@ -1,6 +1,11 @@
+[id='concept-explanation-{context}']
+= Overview of {PRODUCT}
 
-[id='rn-intro-con']
+{PRODUCT} enables your team to develop, build and deploy enterprise integration solutions on a private Kubernetes cluster managed by Red Hat. 
+{PRODUCT} enables developers adopt event-based computing and API-based development with lightweight, loosely coupled, self-service and flexible components. 
+{PRODUCT} enables operations focus on high value objectives because Red Hat manage the infrastructure and everything is protected by RHSSO.
```
Comment (catrobson) [inline docs/content/rn-intro-con.adoc:6]: I would recommend removing this last sentence, I don't think it adds anything but more confusing detail on the "managed by Red Hat" for initial user understanding of the service.  I would expect the layer of detail of what it saves Operations to be captured in a more detailed area of the documentation.

**Pull request closed without merging**